docs/reference.md

```diff
@@ -0,0 +1,206 @@
+# Reference Docs
+
+## Composition Basics
+
+The most basic composition includes only a reference to its synthesizer.
+
+```yaml
+apiVersion: eno.azure.io/v1
+kind: Composition
+metadata:
+  name: example
+spec:
+  synthesizer:
+    name: test-synth
+```
+
+This example will cause a "synthesis" of the composition through its synthesizer
+and track the lifecycle of the resulting resources:
+deleting the composition will delete the resources,
+updating it will cause resynthesis, etc.
+
+### Deletion Modes
+
+To keep any resources created because of a composition,
+Eno supports an orphaning deletion mode by setting this annotation:
```

Comment:
Do you think the phrasing here is clear enough to indicate that the annotation should go on the composition?